### PR TITLE
Clean up tests and CI

### PR DESCRIPTION
**Pull request by aymeric-ledger**

### CODE REVIEW

File: src/test/scala/co/ledger/wallet/daemon/clients/ApiClientTest.scala [hunk 1]
```diff
@@ -6,14 +6,6 @@ import org.scalatest.junit.AssertionsForJUnit
 import scala.concurrent.Await
 import scala.concurrent.duration.Duration
 
-/**
-  * Test cases for [[ApiClient]].
-  *
-  * User: Ting Tu
```
Comment (YBadiss) [inline src/test/scala/co/ledger/wallet/daemon/clients/ApiClientTest.scala]: :'(

File: src/test/resources/logback-test.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<configuration>
```
Comment (luchenyuxx) [inline src/test/resources/logback-test.xml]: Why it's renamed to `logback-test`?

File: project/plugins.sbt [hunk 3]
```diff
@@ -1,2 +1,6 @@
-addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.3.25")
+addCompilerPlugin("org.psywerx.hairyfotr" %% "linter" % "0.1.17")
```
Comment (luchenyuxx) [inline project/plugins.sbt]: The author suggests putting this in `build.sbt`

File: src/test/resources/logback-test.xml (hunk 2 above)
Comment (aymeric-ledger) [reply src/test/resources/logback-test.xml]: I understand this is the standard way of doing it.

**Pull request merged**